variables.tf

```diff
@@ -114,18 +114,18 @@ variable "resource_limits_cpu" {
 
 variable "resource_limits_memory" {
   type        = string
-  default     = "512Mi"
+  default     = "2Gi"
```

Comment:
@osterman @danjbh maybe one of ya'll can comment on if bumping these defaults makes sense or not from your experience with using this module?

Reply:
The current limmits make the agent extremely slow